Moved functionality of entity-children into entity

lloyddave opened the pull request:
Also added aliases for entity command of entities, ent, ents

lloyddave committed:
Moved functionality of entity-children into entity

Also added aliases for entity command of entities, ent, ents

geomacy commented on commands/entity.go:
```diff
@@ -34,7 +35,11 @@ func (cmd *Entity) Run(scope scope.Scope, c *cli.Context) {
 		cmd.show(scope.Application, c.Args().First())
 	} else {
-		cmd.list(scope.Application)
+		if scope.Entity == "" {
```
This should never be true; scope.Entity is defaulted to be equal to scope.Application if you do not explicitly supply an entity scope.

lloyddave replied:
Changed to test on equality of scope.Application and scope.Entity.
Also changed format of show for a single entity to be 2 columns so that it works better on standard width terminals.

lloyddave committed:
Changed to test on equality of scope.Application and scope.Entity.

Changed to test on equality of scope.Application and scope.Entity.
Also changed format of show for a single entity to be 2 columns
so that it works better on standard width terminals.
